main.js

```diff
@@ -160,4 +160,124 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+//Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+const nameOfTvType = inventory.map((tvType) => {
+  return 'tv-type: ' + tvType.type;
+});
+console.log(nameOfTvType);
+
+//Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn. Log de uitkomst in de console.
+const tvOfSold = inventory.filter((soldTv) => {
+  return soldTv.sold > 0;
+    });
```

Comment:
Dit geeft een verkeerd resultaat. Nu krijg je alle tv's waarvan er ten minste één van verkocht is. Je moet kijken of het aantal verkochte tv's gelijk is aan de hoeveelheid tv's die ingekocht zijn:
```suggestion
const tvOfSold = inventory.filter(soldTv => soldTv.sold === soldTv.originalStock);
```